src/utils/validateInputWithRegex.test.tsx

```diff
@@ -1,4 +1,5 @@
 import { validate, validateAndCleanRegexInput } from "./validateInputWithRegex";
+import "../common/i18n/i18nMock";
```

Comment:
I assume this has to be added to every test file which deals with i18n?

Reply:
yup!